api/acl.go

```diff
@@ -22,31 +22,34 @@ type ACLTokenPolicyLink struct {
 
 // ACLToken represents an ACL Token
 type ACLToken struct {
-	CreateIndex uint64
-	ModifyIndex uint64
-	AccessorID  string
-	SecretID    string
-	Description string
-	Policies    []*ACLTokenPolicyLink
-	Local       bool
-	CreateTime  time.Time `json:",omitempty"`
-	Hash        []byte    `json:",omitempty"`
+	CreateIndex    uint64
+	ModifyIndex    uint64
+	AccessorID     string
+	SecretID       string
+	Description    string
+	Policies       []*ACLTokenPolicyLink
+	Local          bool
+	ExpirationTTL  string    `json:",omitempty"`
```

Comment:
Similar to my previous comment about the internal struct. We could just use a time.Duration here instead of a string.

Reply:
Will do.